src/controllers/game.controller.ts

```diff
@@ -56,10 +60,15 @@ export const updateGame = async (req: Request, res: Response) => {
       throw new Error("game not found");
     }
 
+    // not sure if we do this here or in a separate route we call after updating final scores
```

Comment:
Looks good for me here! (but no super strong opinions if you'd rather do it in a second route!)